Rakefile

```diff
@@ -29,7 +29,8 @@ end
 
 desc 'load irb with this gem'
 task :console do
-  exec 'irb -I lib -r coverband'
+  puts 'running console'
+  exec 'bundle exec irb -I lib -r coverband'
```

Comment:
Does this do something different then `bundle console`?

Reply:
nope I hadn't used bundle console... I guess I could just delete this task ;)

Reply:
opposed to removing I updated to bundle console in the rake task cause I would forget again ;)